src/schemes/boundary/open_boundary/system.jl

```diff
@@ -0,0 +1,508 @@
+include("boundary_zones.jl")
+
+"""
+    OpenBoundarySPHSystem(boundary_zone::Union{InFlow, OutFlow}, sound_speed;
+                          buffer_size=0,
+                          reference_velocity=zeros(ndims(boundary_zone)),
+                          reference_pressure=0.0,
+                          reference_density=first(boundary_zone.initial_condition.density))
+
+Open boundary system for in- and outflow particles.
+These open boundaries use the characteristic variables to propagate the appropriate values
+to the outlet or inlet and has been proposed by Lastiwka et al (2009). For more information
+about the method see [Open Boundary System](@ref open_boundary).
+
+# Arguments
+- `boundary_zone`: Use [`InFlow`](@ref) for an inflow and [`OutFlow`](@ref) for an outflow boundary.
+- `sound_speed`: Speed of sound.
+
+# Keywords
+- `buffer_size`: Number of buffer particles.
```

Comment:
Yes, it will also work if  `buffer_size=0`. When I have a stationary flow with no turbulence or relative velocities, all particles leaving the zone will be deactivated and activated at same time. I tested this and it works

Reply:
Okay then. My comment still stands, though.

Reply:
No, it wouldn't work, because activating and deactivating particles is only possible with a `SystemBuffer`

Reply:
But please implement my suggestion above:
```suggestion
- `buffer_size=0`: Number of buffer particles.
```